### DISCUSSION TITLE
how to load all languages one times

### DISCUSSION THREAD

**Discussion by ghost** [PrismJS/prism#2962] [Q&A] [answered]
https://prismjs.com/#supported-languages

languages is to much

```js
import 'prismjs/components/prism-python.min'; // python
import 'prismjs/components/prism-javascript.min'; //js
import 'prismjs/components/prism-css.min'; //css
...
```

how to load all languages one times, like this
```js
prismjs.loadALL_languages()
```

**Comment by RunDevelopment** [comment, marked as answer]
Currently, there are 2 ways of doing this:

1. Go to our [download page](https://prismjs.com/download.html), select all languages, and download the build.

    This will give you all languages and work in all environments (Node, browser, etc.).

2. Use [`loadLanguages`](https://prismjs.com/index.html#basic-usage-node). 

    Calling `loadLanguages()` without arguments will load all languages. However, this is intended to be used with Node.js.

**Reply by ghost** [reply]
thank you very much
